### ISSUE TITLE
compile error on `sbt docker:publishLocal` with latest jdk

### ISSUE THREAD

**Issue by rahulch** [softwaremill/akka-simple-cluster-k8s#3]
### java --version
```
openjdk 13 2019-09-17
OpenJDK Runtime Environment (build 13+33)
OpenJDK 64-Bit Server VM (build 13+33, mixed mode, sharing)
```

### Compile Error
```
[error] Caused by: java.lang.RuntimeException: /packages cannot be represented as URI
[error] 	at java.base/jdk.internal.jrtfs.JrtPath.toUri(JrtPath.java:176)
[error] 	at scala.tools.nsc.classpath.JrtClassPath.asURLs(DirectoryClassPath.scala:183)
[error] 	at scala.tools.nsc.classpath.AggregateClassPath.$anonfun$asURLs$1(AggregateClassPath.scala:55)
[error] 	at scala.collection.TraversableLike.$anonfun$flatMap$1(TraversableLike.scala:241)
[error] 	at scala.collection.Iterator.foreach(Iterator.scala:929)
[error] 	at scala.collection.Iterator.foreach$(Iterator.scala:929)
[error] 	at scala.collection.AbstractIterator.foreach(Iterator.scala:1417)
[error] 	at scala.collection.IterableLike.foreach(IterableLike.scala:71)
[error] 	at scala.collection.IterableLike.foreach$(IterableLike.scala:70)
[error] 	at scala.collection.AbstractIterable.foreach(Iterable.scala:54)
[error] 	at scala.collection.TraversableLike.flatMap(TraversableLike.scala:241)
[error] 	at scala.collection.TraversableLike.flatMap$(TraversableLike.scala:238)
[error] 	at scala.collection.AbstractTraversable.flatMap(Traversable.scala:104)
[error] 	at scala.tools.nsc.classpath.AggregateClassPath.asURLs(AggregateClassPath.scala:55)
[error] 	at scala.tools.nsc.typechecker.Macros.findMacroClassLoader(Macros.scala:62)
[error] 	at scala.tools.nsc.typechecker.Macros.findMacroClassLoader$(Macros.scala:61)
[error] 	at scala.tools.nsc.Global$$anon$1.findMacroClassLoader(Global.scala:458)
[error] 	at scala.reflect.macros.runtime.MacroRuntimes.$anonfun$defaultMacroClassloaderCache$2(MacroRuntimes.scala:56)
[error] 	at scala.reflect.internal.SymbolTable$perRunCaches$$anon$1.apply(SymbolTable.scala:411)
[error] 	at scala.reflect.macros.runtime.MacroRuntimes.defaultMacroClassloader(MacroRuntimes.scala:58)
[error] 	at scala.reflect.macros.runtime.MacroRuntimes.defaultMacroClassloader$(MacroRuntimes.scala:58)
[error] 	at scala.tools.nsc.Global$$anon$1.defaultMacroClassloader(Global.scala:458)
[error] 	at scala.reflect.macros.runtime.MacroRuntimes$MacroRuntimeResolver.resolveRuntime(MacroRuntimes.scala:76)
[error] 	at scala.reflect.macros.runtime.MacroRuntimes.$anonfun$standardMacroRuntime$3(MacroRuntimes.scala:38)
[error] 	at scala.collection.mutable.MapLike.getOrElseUpdate(MapLike.scala:206)
[error] 	at scala.collection.mutable.MapLike.getOrElseUpdate$(MapLike.scala:203)
[error] 	at scala.collection.mutable.AbstractMap.getOrElseUpdate(Map.scala:80)
[error] 	at scala.reflect.macros.runtime.MacroRuntimes.standardMacroRuntime(MacroRuntimes.scala:38)
[error] 	at scala.reflect.macros.runtime.MacroRuntimes.standardMacroRuntime$(MacroRuntimes.scala:31)
[error] 	at scala.tools.nsc.Global$$anon$1.standardMacroRuntime(Global.scala:458)
[error] 	at scala.tools.nsc.typechecker.AnalyzerPlugins$$anon$14.default(AnalyzerPlugins.scala:448)
[error] 	at scala.tools.nsc.typechecker.AnalyzerPlugins$$anon$14.default(AnalyzerPlugins.scala:445)
[error] 	at scala.tools.nsc.typechecker.AnalyzerPlugins.invoke(AnalyzerPlugins.scala:401)
[error] 	at scala.tools.nsc.typechecker.AnalyzerPlugins.pluginsMacroRuntime(AnalyzerPlugins.scala:445)
[error] 	at scala.tools.nsc.typechecker.AnalyzerPlugins.pluginsMacroRuntime$(AnalyzerPlugins.scala:445)
[error] 	at scala.tools.nsc.Global$$anon$1.pluginsMacroRuntime(Global.scala:458)
[error] 	at scala.reflect.macros.runtime.MacroRuntimes.macroRuntime(MacroRuntimes.scala:25)
[error] 	at scala.reflect.macros.runtime.MacroRuntimes.macroRuntime$(MacroRuntimes.scala:25)
[error] 	at scala.tools.nsc.Global$$anon$1.macroRuntime(Global.scala:458)
[error] 	at scala.tools.nsc.typechecker.Macros$MacroExpander.$anonfun$expand$1(Macros.scala:588)
[error] 	at scala.tools.nsc.Global.withInfoLevel(Global.scala:208)
[error] 	at scala.tools.nsc.typechecker.Macros$MacroExpander.expand(Macros.scala:582)
[error] 	at scala.tools.nsc.typechecker.Macros$MacroExpander.apply(Macros.scala:569)
[error] 	at scala.tools.nsc.typechecker.Macros.standardMacroExpand(Macros.scala:733)
[error] 	at scala.tools.nsc.typechecker.Macros.standardMacroExpand$(Macros.scala:731)
[error] 	at scala.tools.nsc.Global$$anon$1.standardMacroExpand(Global.scala:458)
[error] 	at scala.tools.nsc.typechecker.AnalyzerPlugins$$anon$12.default(AnalyzerPlugins.scala:432)
[error] 	at scala.tools.nsc.typechecker.AnalyzerPlugins$$anon$12.default(AnalyzerPlugins.scala:429)
[error] 	at scala.tools.nsc.typechecker.AnalyzerPlugins.invoke(AnalyzerPlugins.scala:401)
[error] 	at scala.tools.nsc.typechecker.AnalyzerPlugins.pluginsMacroExpand(AnalyzerPlugins.scala:429)
[error] 	at scala.tools.nsc.typechecker.AnalyzerPlugins.pluginsMacroExpand$(AnalyzerPlugins.scala:429)
[error] 	at scala.tools.nsc.Global$$anon$1.pluginsMacroExpand(Global.scala:458)
[error] 	at scala.tools.nsc.typechecker.Macros.macroExpand(Macros.scala:726)
[error] 	at scala.tools.nsc.typechecker.Macros.macroExpand$(Macros.scala:726)
[error] 	at scala.tools.nsc.Global$$anon$1.macroExpand(Global.scala:458)
[error] 	at scala.tools.nsc.typechecker.Typers$Typer.vanillaAdapt$1(Typers.scala:1154)
[error] 	at scala.tools.nsc.typechecker.Typers$Typer.adapt(Typers.scala:1214)
[error] 	at scala.tools.nsc.typechecker.Typers$Typer.runTyper$1(Typers.scala:5598)
[error] 	at scala.tools.nsc.typechecker.Typers$Typer.typedInternal(Typers.scala:5616)
[error] 	at scala.tools.nsc.typechecker.Typers$Typer.body$2(Typers.scala:5557)
[error] 	at scala.tools.nsc.typechecker.Typers$Typer.typed(Typers.scala:5562)
[error] 	at scala.tools.nsc.typechecker.Typers$Typer.transformedOrTyped(Typers.scala:5798)
[error] 	at scala.tools.nsc.typechecker.Typers$Typer.typedDefDef(Typers.scala:2287)
[error] 	at scala.tools.nsc.typechecker.Typers$Typer.typedMemberDef$1(Typers.scala:5495)
[error] 	at scala.tools.nsc.typechecker.Typers$Typer.typed1(Typers.scala:5546)
[error] 	at scala.tools.nsc.typechecker.Typers$Typer.runTyper$1(Typers.scala:5584)
[error] 	at scala.tools.nsc.typechecker.Typers$Typer.typedInternal(Typers.scala:5616)
[error] 	at scala.tools.nsc.typechecker.Typers$Typer.body$2(Typers.scala:5557)
[error] 	at scala.tools.nsc.typechecker.Typers$Typer.typed(Typers.scala:5562)
[error] 	at scala.tools.nsc.typechecker.Typers$Typer.typedByValueExpr(Typers.scala:5647)
[error] 	at scala.tools.nsc.typechecker.Typers$Typer.typedStat$1(Typers.scala:3086)
[error] 	at scala.tools.nsc.typechecker.Typers$Typer.$anonfun$typedStats$10(Typers.scala:3231)
[error] 	at scala.tools.nsc.typechecker.Typers$Typer.typedStats(Typers.scala:3231)
[error] 	at scala.tools.nsc.typechecker.Typers$Typer.typedTemplate(Typers.scala:1987)
[error] 	at scala.tools.nsc.typechecker.Typers$Typer.typedModuleDef(Typers.scala:1858)
[error] 	at scala.tools.nsc.typechecker.Typers$Typer.typedMemberDef$1(Typers.scala:5497)
[error] 	at scala.tools.nsc.typechecker.Typers$Typer.typed1(Typers.scala:5546)
[error] 	at scala.tools.nsc.typechecker.Typers$Typer.runTyper$1(Typers.scala:5584)
[error] 	at scala.tools.nsc.typechecker.Typers$Typer.typedInternal(Typers.scala:5616)
[error] 	at scala.tools.nsc.typechecker.Typers$Typer.body$2(Typers.scala:5557)
[error] 	at scala.tools.nsc.typechecker.Typers$Typer.typed(Typers.scala:5562)
[error] 	at scala.tools.nsc.typechecker.Typers$Typer.typedByValueExpr(Typers.scala:5647)
[error] 	at scala.tools.nsc.typechecker.Typers$Typer.typedStat$1(Typers.scala:3086)
[error] 	at scala.tools.nsc.typechecker.Typers$Typer.$anonfun$typedStats$10(Typers.scala:3231)
[error] 	at scala.tools.nsc.typechecker.Typers$Typer.typedStats(Typers.scala:3231)
[error] 	at scala.tools.nsc.typechecker.Typers$Typer.typedPackageDef$1(Typers.scala:5207)
[error] 	at scala.tools.nsc.typechecker.Typers$Typer.typedMemberDef$1(Typers.scala:5499)
[error] 	at scala.tools.nsc.typechecker.Typers$Typer.typed1(Typers.scala:5546)
[error] 	at scala.tools.nsc.typechecker.Typers$Typer.runTyper$1(Typers.scala:5584)
[error] 	at scala.tools.nsc.typechecker.Typers$Typer.typedInternal(Typers.scala:5616)
[error] 	at scala.tools.nsc.typechecker.Typers$Typer.body$2(Typers.scala:5557)
[error] 	at scala.tools.nsc.typechecker.Typers$Typer.typed(Typers.scala:5562)
[error] 	at scala.tools.nsc.typechecker.Typers$Typer.typed(Typers.scala:5643)
[error] 	at scala.tools.nsc.typechecker.Analyzer$typerFactory$$anon$3.apply(Analyzer.scala:104)
[error] 	at scala.tools.nsc.Global$GlobalPhase.$anonfun$applyPhase$1(Global.scala:436)
[error] 	at scala.tools.nsc.Global$GlobalPhase.applyPhase(Global.scala:429)
[error] 	at scala.tools.nsc.typechecker.Analyzer$typerFactory$$anon$3.$anonfun$run$1(Analyzer.scala:94)
[error] 	at scala.tools.nsc.typechecker.Analyzer$typerFactory$$anon$3.$anonfun$run$1$adapted(Analyzer.scala:93)
[error] 	at scala.collection.Iterator.foreach(Iterator.scala:929)
[error] 	at scala.collection.Iterator.foreach$(Iterator.scala:929)
[error] 	at scala.collection.AbstractIterator.foreach(Iterator.scala:1417)
[error] 	at scala.tools.nsc.typechecker.Analyzer$typerFactory$$anon$3.run(Analyzer.scala:93)
[error] 	at sbt.compiler.Eval.$anonfun$compileAndLoad$1(Eval.scala:214)
[error] 	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
[error] 	at scala.reflect.internal.SymbolTable.enteringPhase(SymbolTable.scala:250)
[error] 	at sbt.compiler.Eval.compile$1(Eval.scala:214)
[error] 	at sbt.compiler.Eval.compileAndLoad(Eval.scala:219)
[error] 	at sbt.compiler.Eval.evalCommon(Eval.scala:186)
[error] 	at sbt.compiler.Eval.eval(Eval.scala:119)
[error] 	at sbt.internal.EvaluateConfigurations$.evaluateDslEntry(EvaluateConfigurations.scala:180)
[error] 	at sbt.internal.EvaluateConfigurations$.$anonfun$evaluateSbtFile$2(EvaluateConfigurations.scala:120)
[error] 	at scala.collection.TraversableLike.$anonfun$map$1(TraversableLike.scala:234)
[error] 	at scala.collection.immutable.List.foreach(List.scala:389)
[error] 	at scala.collection.TraversableLike.map(TraversableLike.scala:234)
[error] 	at scala.collection.TraversableLike.map$(TraversableLike.scala:227)
[error] 	at scala.collection.immutable.List.map(List.scala:295)
[error] 	at sbt.internal.EvaluateConfigurations$.evaluateSbtFile(EvaluateConfigurations.scala:118)
[error] 	at sbt.internal.Load$.loadSettingsFile$1(Load.scala:1087)
[error] 	at sbt.internal.Load$.$anonfun$discoverProjects$2(Load.scala:1095)
[error] 	at scala.collection.MapLike.getOrElse(MapLike.scala:128)
[error] 	at scala.collection.MapLike.getOrElse$(MapLike.scala:126)
[error] 	at scala.collection.AbstractMap.getOrElse(Map.scala:59)
[error] 	at sbt.internal.Load$.memoLoadSettingsFile$1(Load.scala:1094)
[error] 	at sbt.internal.Load$.$anonfun$discoverProjects$4(Load.scala:1102)
[error] 	at scala.collection.TraversableLike.$anonfun$map$1(TraversableLike.scala:234)
[error] 	at scala.collection.mutable.ResizableArray.foreach(ResizableArray.scala:59)
[error] 	at scala.collection.mutable.ResizableArray.foreach$(ResizableArray.scala:52)
[error] 	at scala.collection.mutable.ArrayBuffer.foreach(ArrayBuffer.scala:48)
[error] 	at scala.collection.TraversableLike.map(TraversableLike.scala:234)
[error] 	at scala.collection.TraversableLike.map$(TraversableLike.scala:227)
[error] 	at scala.collection.AbstractTraversable.map(Traversable.scala:104)
[error] 	at sbt.internal.Load$.loadFiles$1(Load.scala:1102)
[error] 	at sbt.internal.Load$.discoverProjects(Load.scala:1116)
[error] 	at sbt.internal.Load$.discover$1(Load.scala:844)
[error] 	at sbt.internal.Load$.loadTransitive(Load.scala:909)
[error] 	at sbt.internal.Load$.loadProjects$1(Load.scala:714)
[error] 	at sbt.internal.Load$.$anonfun$loadUnit$10(Load.scala:717)
[error] 	at sbt.internal.Load$.timed(Load.scala:1343)
[error] 	at sbt.internal.Load$.$anonfun$loadUnit$1(Load.scala:717)
[error] 	at sbt.internal.Load$.timed(Load.scala:1343)
[error] 	at sbt.internal.Load$.loadUnit(Load.scala:677)
[error] 	at sbt.internal.Load$.$anonfun$builtinLoader$4(Load.scala:477)
[error] 	at sbt.internal.BuildLoader$.$anonfun$componentLoader$5(BuildLoader.scala:158)
[error] 	at sbt.internal.BuildLoader.apply(BuildLoader.scala:223)
[error] 	at sbt.internal.Load$.loadURI$1(Load.scala:539)
[error] 	at sbt.internal.Load$.loadAll(Load.scala:555)
[error] 	at sbt.internal.Load$.loadURI(Load.scala:485)
[error] 	at sbt.internal.Load$.load(Load.scala:464)
[error] 	at sbt.internal.Load$.$anonfun$apply$1(Load.scala:253)
[error] 	at sbt.internal.Load$.timed(Load.scala:1343)
[error] 	at sbt.internal.Load$.apply(Load.scala:253)
[error] 	at sbt.internal.Load$.defaultLoad(Load.scala:69)
[error] 	at sbt.BuiltinCommands$.liftedTree1$1(Main.scala:707)
[error] 	at sbt.BuiltinCommands$.doLoadProject(Main.scala:707)
[error] 	at sbt.BuiltinCommands$.$anonfun$loadProjectImpl$2(Main.scala:680)
[error] 	at sbt.Command$.$anonfun$applyEffect$4(Command.scala:134)
[error] 	at sbt.Command$.$anonfun$applyEffect$2(Command.scala:130)
[error] 	at sbt.MainLoop$.processCommand(MainLoop.scala:153)
[error] 	at sbt.MainLoop$.$anonfun$next$2(MainLoop.scala:136)
[error] 	at sbt.State$$anon$1.runCmd$1(State.scala:242)
[error] 	at sbt.State$$anon$1.process(State.scala:248)
[error] 	at sbt.MainLoop$.$anonfun$next$1(MainLoop.scala:136)
[error] 	at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:16)
[error] 	at sbt.MainLoop$.next(MainLoop.scala:136)
[error] 	at sbt.MainLoop$.run(MainLoop.scala:129)
[error] 	at sbt.MainLoop$.$anonfun$runWithNewLog$1(MainLoop.scala:107)
[error] 	at sbt.io.Using.apply(Using.scala:22)
[error] 	at sbt.MainLoop$.runWithNewLog(MainLoop.scala:101)
[error] 	at sbt.MainLoop$.runAndClearLast(MainLoop.scala:57)
[error] 	at sbt.MainLoop$.runLoggedLoop(MainLoop.scala:42)
[error] 	at sbt.MainLoop$.runLogged(MainLoop.scala:34)
[error] 	at sbt.StandardMain$.runManaged(Main.scala:113)
[error] 	at sbt.xMain.run(Main.scala:76)
[error] 	at xsbt.boot.Launch$$anonfun$run$1.apply(Launch.scala:111)
[error] 	at xsbt.boot.Launch$.withContextLoader(Launch.scala:130)
[error] 	at xsbt.boot.Launch$.run(Launch.scala:111)
[error] 	at xsbt.boot.Launch$$anonfun$apply$1.apply(Launch.scala:37)
[error] 	at xsbt.boot.Launch$.launch(Launch.scala:119)
[error] 	at xsbt.boot.Launch$.apply(Launch.scala:20)
[error] 	at xsbt.boot.Boot$.runImpl(Boot.scala:56)
[error] 	at xsbt.boot.Boot$.main(Boot.scala:18)
[error] 	at xsbt.boot.Boot.main(Boot.scala)
[error] java.io.IOError: java.lang.RuntimeException: /packages cannot be represented as URI
[error] Use 'last' for the full log.

```

**Comment by rahul-coatue** [comment]
I had to change the sbt version to fix this. `sbt 1.3.2` works fine

**Comment by rahulch** [comment]
This got fixed when I upgraded the sbt version to 1.3.2
